cmd/template/dbdriver/files/service/redis.tmpl

```diff
@@ -40,31 +41,156 @@ func New() Service {
 		Addr:     fullAddress,
 		Password: password,
 		DB:       num,
-		TLSConfig: &tls.Config{
-			MinVersion: tls.VersionTLS12,
-		},
+		// TLSConfig: &tls.Config{
+		// 	MinVersion:   tls.VersionTLS12,
+		// },
 	})
```

Comment:
Your comments regarding TLS were useful and could be included as part of the documentation.